operator/internal/controller/podcliquescalinggroup/reconcilestatus.go

```diff
@@ -23,8 +23,8 @@ import (
 	apicommon "github.com/NVIDIA/grove/operator/api/common"
 	"github.com/NVIDIA/grove/operator/api/common/constants"
 	grovecorev1alpha1 "github.com/NVIDIA/grove/operator/api/core/v1alpha1"
-	componentutils "github.com/NVIDIA/grove/operator/internal/component/utils"
 	ctrlcommon "github.com/NVIDIA/grove/operator/internal/controller/common"
+	"github.com/NVIDIA/grove/operator/internal/controller/common/component/utils"
```

Comment:
To be consistent with the rest of the codebase.

```suggestion
	componentutils "github.com/NVIDIA/grove/operator/internal/controller/common/component/utils"
```